Realm/RLMAccessor.mm

```diff
@@ -648,19 +682,19 @@ void RLMDynamicSet(__unsafe_unretained RLMObjectBase *const obj, __unsafe_unreta
                 RLMSetValue(obj, col, (NSString *)val);
             }
             break;
-        case 'a':
+        case RLMAccessorCodeDate:
             RLMSetValue(obj, col, (NSDate *)val);
             break;
-        case 'e':
+        case RLMAccessorCodeData:
             RLMSetValue(obj, col, (NSData *)val);
             break;
-        case 'k':
+        case RLMAccessorCodeLink:
             RLMSetValue(obj, col, (RLMObjectBase *)val, options);
             break;
-        case 't':
+        case RLMAccessorCodeArray:
             RLMSetValue(obj, col, (RLMArray *)val, options);
             break;
-        case '@':
+        case RLMAccessorCodeAny:
             RLMSetValue(obj, col, val);
             break;
         default:
```

Comment:
Should be able to remove the `default:` cases from most of the places switching on the enum.